documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,200 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+
+* Starting Pravega standalone.
+* Creating a Pravega Stream.
+* Creating an Event Writer and write events into a Pravega Stream.
+* Create an Event Reader and read events from the Pravega Stream.
+
+ 
+# 1. Prerequisites
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Gradle 6.5.1+
+
+<details open>
```

Comment:
Looks good, but it is the first time I see this in Pravega documentation.

Reply:
i have improved formatting here.